examples/QuickStart3_0/Readme.md

```diff
@@ -7,7 +7,7 @@ Then, in the second part, we will containerized it to a Linux container.
 ## Prerequisites
 
 1. [.NET Core **SDK** 3.0.100](https://dotnet.microsoft.com/download).
-1. [An instrumentation key](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource) associated with a valid application insights resource in Azure.
+1. [A connection string](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource) associated with a valid application insights resource in Azure.
```

Comment:
What about adding this link too? https://learn.microsoft.com/en-us/azure/azure-monitor/app/migrate-from-instrumentation-keys-to-connection-strings#migrate-from-application-insights-instrumentation-keys-to-connection-strings